runtime/tool/purge.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tool
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"text/template"
+)
+
+// PurgeSpec configures the command returned by PurgeCmd.
+type PurgeSpec struct {
```

Comment:
Maybe add a TODO here; probably we will also specify some reqular expression or smthg in the `PurgeSpec` based on which we will kill all processes/subprocesses for each deployer?

Reply:
Done. Good idea.